src/index/formatFileStructure/shared/findEdges.ts

```diff
@@ -15,7 +20,9 @@ export function findEdges(filePath: string) {
     if (matches.index === importRegex.lastIndex) {
       importRegex.lastIndex++;
     }
-    edges.push([filePath, matches[1] || matches[2]]);
+    const importMatch = matches[1];
+    const requireMatch = matches[2];
+    edges.push([filePath, importMatch || requireMatch]);
```

Comment:
tips: we can use `??` operator with latest typescript version

Reply:
Okay thank you.